shell/server/pre-meteor.js

```diff
@@ -452,9 +460,9 @@ Meteor.startup(() => {
         return;
       }
 
-      if (id === 'static') {
+      if (id === "static" || id === "identicon") {
```

Comment:
There's an argument to be made that we should differentiate these by path rather than by host: When a browser sees a new hostname, it is generally forced to create a new HTTP/HTTPS connection to that host rather than reuse an existing one, hurting performance.